sdk/tests/conformance2/renderbuffers/framebuffer-texture-layer.html

```diff
@@ -105,6 +105,7 @@
     gl.framebufferTextureLayer(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, tex3d, maxLevels - 1, 0);
     wtu.glErrorShouldBe(gl, gl.NO_ERROR,
         "calling framebufferTextureLayer with an appropriate mipmap level should succeed.");
+    checkFramebuffer([gl.FRAMEBUFFER_INCOMPLETE_ATTACHMENT]);
```

Comment:
The attached texture failed completeness check due to texture image doesn't exist in (maxLevels -1) level.